causing/examples/models.py

```diff
@@ -201,7 +209,7 @@ def education():
     # load and transform data
     from numpy import array, concatenate, exp, loadtxt
 
-    xymdat = loadtxt("data/education.csv", delimiter=",").reshape(-1, 10)
+    xymdat = loadtxt("{}education.csv".format(DATA_PATH), delimiter=",").reshape(-1, 10)
```

Comment:
We know exactly where the files are inside this repository, so why do we need an environment variable to configure the location?

The previous code worked fine when running `python -m causing.examples education` for the root of the repo. If we want to allow running the example from anywhere, we could find out where the file is by starting at `__file__`(the current python modules path) and specifying the known relative path from there. But do we really need that?

You might have noticed that a CI job failed. This is because running `python -m causing.examples education` does not work anymore without modifying the DATA_PATH.